eclipse-collections/src/main/java/org/eclipse/collections/impl/multimap/AbstractSynchronizedMultimap.java

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2016 Shotaro Sano.
+ * Copyright (c) 2018 Shotaro Sano and others.
```

Comment:
Copyright-only change to this file. Is it intentional?

Reply:
I pushed it to implementations due to covariant overrides, forgot to undo the copyright update. Fixed.